### PR TITLE
Issue 5048 - Reacting to issues and risks being disabled or enabled

### LINKED ISSUE #5048: Detect optional features and disable risks/issues if they are not available in the TeamSpace
### Description
Product issue: https://github.com/3drepo/3D-Repo-Product-Team/issues/530
We are providing an option to enable/disabled issues and risks based on the TeamSpace.
This depends on #5046

 
- If issues is not enabled frontend will hide issues in the viewer.
- If risks is not enabled frontend will hide risks in the viewer.
- If issues is not enabled kanban will hide the issues options in the select box
- If risks is not enabled kanban will hide the risks options in the select box
- if issues and risks are not enable the whole kanban route will not be enabled

<details>
   <summary> Backend endpoints </summary>

Route: `HTTP GET /v5/teamspaces/{teamspace}/addOns`

```json
{
    "vrEnabled" : true,
    "hereEnabled" : true,
    "modules": ["issues", "risks"]
}
```
</details>

### Goals
- [ ] As a teamspace/project admin/user, when a system admin turns off Issues or Risks in the teamspace settings I should not see this on the 3D Repo.io UI.

### Tasks
- [x] Add API functions for getting the addons
- [x] Add redux states/sagas for getting the addons
- [x] Fetch the addons state
- [x] Remove issues from the viewer if they are not enabled
- [x] Remove risks from the viewer if they are not enabled
- [x] Remove issues from the kanban select box if they are not enabled
- [x] Remove risks from the kanban select box if they are not enabled
- [x] Change title of issues & risks tab based on the state of addons
- [x] Remove kanban route if risks and issues are not enabled

### PR DESCRIPTION
**Pull request by sanmont3drepo**
This fixes #5048 

#### Description
- This branch is to react to issues and risks being optional modules
- If risks are not enabled the risks button/card cant be accessed from the viewer and the pins/shapes are not being rendered
- If issues are  not enabled the issues button/card cant be accessed from the viewer and the pins/shapes are not being rendered
- If only issues are enabled the kanban board only shows issues
- if only risks are enabled the kanban board only shows risks
- The kanban board is not being rendered nor link is  being supplied if issues and risks are disabled
- ATM the modules defitions are being mocked in `src\v5\services\api\teamspaces.ts`

#### Test cases
- Test the full functionality with issues  and risks enabled
- Test the full functionality with just issues enabled
- Test the full functionality with just risks enabled
- Test the full functionality with risks and risks disabled

### CODE REVIEW

File: frontend/src/v4/modules/issues/issues.sagas.ts [hunk 1]
```diff
@@ -80,6 +90,14 @@ function* fetchIssues({teamspace, modelId, revision}) {
 }
 
 function* fetchIssue({teamspace, modelId, issueId}) {
+	yield waitForAddons();
+	const issuesEnabled = yield select(selectIssuesEnabled);
+
+	if (!issuesEnabled) {
```
Comment (The-Daniel) [inline frontend/src/v4/modules/issues/issues.sagas.ts:96]: Is this needed for this saga? At what point in the app do we call `fetchIssue()` that hasn't already been disabled by the check in `fetchIssues()`?

**Review by The-Daniel — CHANGES_REQUESTED** [review]

File: frontend/src/v4/modules/issues/issues.sagas.ts (hunk 1 above)
Comment (sanmont3drepo) [reply frontend/src/v4/modules/issues/issues.sagas.ts:96]: yeah, but it doesnt hurt to have it there also.

**Review by carmenfan — CHANGES_REQUESTED** [review]
@ChristosTsiotsias  got the backend done and is on a PR https://github.com/3drepo/3drepo.io/pull/5057

Can you guys rebase and hook it up properly?

**Comment by carmenfan** [comment]
@sanmont3drepo  front end tests is failing here

**Review by carmenfan — APPROVED** [review]

**Review by The-Daniel — APPROVED** [review]

**Pull request merged**